.github/workflows/node.js.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     if: "!contains(github.event.head_commit.message, '[ci skip]')"
     strategy:
       matrix:
-        node-version: [10.16, 10.19, 12.16, 12.19, 14.16, 14.18]
+        node-version: [10.16, 10.19, 12.16, 12.19, 14.16, 14.18, 16.13]
```

Comment:
looks like a test is failing for node 16: https://github.com/adobe/node-httptransfer/runs/4721578604?check_suite_focus=true
do we claim to support node 16 or need to?

Reply:
We don't claim yet, but node 16 is LTS now

Reply:
(Also I think the failure for node16 was just a fluke)